docs/understanding-your-pipeline/data-products/index.md

```diff
@@ -0,0 +1,81 @@
+* Define the entities to attach to those events (i.e. user, product etc) 
+* Easily communicate tracking requirements to developers implementing tracking
+* Provide important metadata to make the semantics of the data well understood across your business 
+* Subscribe to the data product to receive notifications when changes are being made
+* View a change log to understand how the data has changed over time
+
+
+## Key elements of a Data Product
+
+**An owner**; data products are typically split by domain with each data product having an explicit owner that is responsible for the maintenance and evolution of that data.
+
+**Benefits:**
+
+* Know who to go to when new data is required for a particular domain or when there is an issue with the data, breaking down the barriers between data producers and data consumers
+
+**Event specifications**; these describe each event that is collected as part of the data product, on which applications they are triggered and where, the event data structure to validate against, and the entities to attach to each event (i.e. user, product etc).
+
+**Benefits:**
+
+* Provide implementation details to developers implementing tracking (see section on [Snowtype](/docs/collecting-data/code-generation) for further details)
+* Provide documentation around the semantics of the data that you are creating, to enable analysts, data scientists, analytics engineers with data discoverability
+
+**Subscribers;** these describe people who are interested in this Data product or who are actively using the data in other ways.
+
+**Benefits:**
+
+* Provide notifications for changes in a Data product.
+
+**Change history**; a complete audit log of the changes that have been made to the data product, including changes to existing data and new data that has been added over time.
+
+**Benefits:**
+
+* Enhances accountability and transparency by providing a clear audit trail of all data modifications, fostering confidence in data integrity
+
+## How data products help with governance, data quality and data discoverability
+
+The **data structures** that you attach to your event specification describe the [structure of the data](/docs/understanding-your-pipeline/schemas/). They validate that the values of the properties contained within your events and entities are **valid** as they pass through your pipeline.
+
+The **event specification** describes a specific implementation of an event. It is a narrower definition of your event than a data structure - not only do they describe the structure of the data (by attaching the relevant data structure to validate against), they also allow you to define the **right values** for fields when the event gets triggered and the entities that need to be attached.
+
+*NB: the event specifications are not enforced by your pipeline (for example, we don’t yet validate that the correct entities are attached to an event) - this is coming soon!*
```

Comment:
I'm fine with the disclaimer here, but a bit uncomfortable with the promise that we're working on it (we're not). @ebedford420 ?

Reply:
I agree with you, I removed the text for now, @ebedford420 in case you would like to re-phrase it please let me know